src/aws/fetchSignedUrl.ts

```
@@ -0,0 +1,41 @@
+      ACL: 'public-read',
+    })
+
+    const signedUrl = await getSignedUrl(client, command, {
+      expiresIn: payload.expiresIn ?? 60,
+    })
+
+    success = true
+    data = signedUrl
+  } catch (error) {
+    success = false
+    error = `fetchSignedUrl failed (${payload.bucket}/${payload.key}): ${
```

Biome (1.9.4)

[error] Reassigning a catch parameter is confusing.

The catch parameter is declared here:

Use a local variable instead.

(lint/suspicious/noCatchAssign)